pkg/features/features.go

```diff
@@ -129,7 +130,7 @@ var AllFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	NodeAgentAuthorizer:                      {Default: false, PreRelease: featuregate.Alpha},
 	CredentialsRotationWithoutWorkersRollout: {Default: false, PreRelease: featuregate.Alpha},
 	InPlaceNodeUpdates:                       {Default: false, PreRelease: featuregate.Alpha},
-	RemoveAPIServerProxyLegacyPort:           {Default: false, PreRelease: featuregate.Alpha},
+	RemoveAPIServerProxyLegacyPort:           {Default: false, PreRelease: featuregate.Beta},
```

Comment:
The release note says that it is enable by default but here in the code it is not enabled by default.

To enable it by default:
```suggestion
	RemoveAPIServerProxyLegacyPort:           {Default: true, PreRelease: featuregate.Beta},
```

Reply:
oh god.... really sorry about that 😓